app/helpers/view_components/example_iterations_summary_table.rb

```diff
@@ -0,0 +1,14 @@
+module ViewComponents
+  class ExampleIterationsSummaryTable < ViewComponent
+    include Mandate
```

Comment:
You should just be able to do this in ViewComponent and not in each individual one I think.

Reply:
Yep